docs/api/exchange_rates.rst

```diff
@@ -0,0 +1,455 @@
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 201 CREATED
+
+.. code-block:: json
+
+    {
+        "id":1,
+        "ratio":0.85157,
+        "sourceCurrency":{
+            "id":2,
+            "code":"EUR",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/currencies\/EUR"
+                }
+            }
+        },
+        "targetCurrency":{
+            "id":3,
+            "code":"GBP",
+            "_links":{
+                "self":{
+                    "href":"\/api\/v1\/currencies\/GBP"
+                }
+            }
+        },
+        "updatedAt":"2017-02-23T15:00:53+0100",
+        "_links":{
+            "self":{
+                "href":"\/api\/v1\/exchange-rates\/EUR-GBP"
+            }
+        }
+    }
+
+If you try to create an exchange rate without required fields you will receive a ``400 Bad Request`` error.
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.orgg/api/v1/exchange-rates/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json" \
+        -X POST
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 400 Bad Request
+
+.. code-block:: json
+
+    {
+        "code":400,
+        "message":"Validation Failed",
+        "errors":{
+            "errors":[
+                "The source and target currencies must differ."
+            ],
+            "children":{
+                "ratio":{
+                    "errors":[
+                        "Please enter exchange rate ratio."
+                    ]
+                },
+                "sourceCurrency":{
+                    "errors":[
+                        "This value is not valid."
+                    ]
+                },
+                "targetCurrency":{
+                    "errors":[
+                        "This value is not valid."
+                    ]
+                }
+            }
+        }
+    }
+
+Getting a Single Exchange Rate
+------------------------------
+
+To retrieve the details of an exchange rate you will need to call the ``/api/v1/exchange-rates/{firstCurrencyCode}-{secondCurrencyCode}`` endpoint with the ``GET`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/exchange-rates/{firstCurrencyCode}-{secondCurrencyCode}
+
++--------------------+----------------+--------------------------------------+
+| Parameter          | Parameter type | Description                          |
++====================+================+======================================+
+| Authorization      | header         | Token received during authentication |
++--------------------+----------------+--------------------------------------+
+| firstCurrencyCode  | url attribute  | First code of currency               |
++--------------------+----------------+--------------------------------------+
+| secondCurrencyCode | url attribute  | Second code of currency              |
```

Comment:
Why is it `first/second` here, but `source/target` [here](https://github.com/Sylius/Sylius/pull/7594/files#diff-0f0e4412efce4f81b023b121d92babbfR52)?